src/main.py

```diff
@@ -63,10 +63,9 @@ def main():
 
         cv2.imshow('frame', frame)
 
-        if cv2.waitKey(1) & 0xFF == ord('q'):
-            break
-    cap.release()
```

Comment:
Why did you remove this??
Isn't it necessary to stop the video capture?

Reply:
it is error message that is causing it to end